packages/react/src/components/Typography/Paragraph/Paragraph.tsx

```diff
@@ -12,8 +12,12 @@ export type ParagraphProps = {
   size?: 'xsmall' | 'small' | 'medium' | 'large';
   /** Adds margin-bottom */
   spacing?: boolean;
-  /** Reduces line-height for short paragraphs */
+  /** Reduces line-height for short paragraphs
+   * @deprecated Use `variant="short"` instead
+   */
   short?: boolean;
+  /** Variant of the paragraph */
+  variant?: 'long' | 'short';
```

Comment:
Should we maybe add `default` so it matches the behaviour of our other `variant` props? It won't do anything but would make life easier for projects like Studio where user can select these props. 🤔

Reply:
hm not sure 🤔 It won't hurt to add it, since it does not do anything - your call 😃

Reply:
We decided to go for `variant`.

Reply:
In reality we have three different line height. We didn't add a `default` option in `variant` as the "default" line-height is backed into the variable used for `font` and it might be changed with static typography. We will add the third option later when the dust has settled for that. [#1707](https://github.com/digdir/designsystemet/issues/1707)